frigate/app.py

```diff
@@ -437,6 +437,7 @@ def start_audio_processors(self) -> None:
             )
             audio_process.daemon = True
             audio_process.start()
+            self.processes["audio"] = audio_process.pid or 0
```

Comment:
```suggestion
            self.processes["audio_detector"] = audio_process.pid or 0
```

Reply:
in audio.py it named "audio_manager", at app.py some lines above it named "audio_capture', and now you suggest name 'audio_detector' for same process. Maybe it's worth deciding? :))

Reply:
Also, this name displayed as card header at System page. IMHO, "audio_detector" isn't good header :)

Reply:
That's why I suggested changing it because `audio`will likely confuse some users as being for recordings audio, so want to make it clear it's the process for the audio detection.

That being said, I think audio_capture can be used in all cases and would be clear enough